identity-and-access-management/audit-trail/quickstart.mdx

```diff
@@ -0,0 +1,52 @@
+---
+meta:
+  title: Audit Trail - Quickstart
+  description: Quickly set up and start using Scaleway’s Audit Trail. Follow steps to configure permissions and track events across regions and projects.
+content:
+  h1: Audit Trail - Quickstart
+  paragraph: Quickly set up and start using Scaleway’s Audit Trail. Follow steps to configure permissions and track events across regions and projects.
+dates:
+  validation: 2024-11-26
+  posted: 2024-11-26
+---
+
+In this quickstart, we show you how to start using [Audit Trail](/identity-and-access-management/audit-trail/concepts/#audit-trail).
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- Created resources integrated with Audit Trail
+
+## How to configure the Audit Trail permission set
+
+To start using Audit Trail you need to configure the `AuditTrailReadOnly` permission set in [IAM](/identity-and-access-management/iam/concepts/#iam). The [scope](/identity-and-access-management/iam/concepts/#scope) of this permission set is at [Organization](/identity-and-access-management/iam/concepts/#organization) level.
```

Comment:
I think we should redirect to the dedicated page about IAM+Audit Trail, because in fact there are other permission sets that can work, + if we need to change info on that better to change it at a single place